### DISCUSSION TITLE
Ability to call different config.json files

### DISCUSSION THREAD

**Discussion by BEisem** [Taxel/PlexTraktSync#190] [Q&A] [answered]
I'm finding that I would like to run a "TV only" instance of the script much more frequently than a full scan. In our house, we watch a lot more TV than we do movies, so I might only want to run a scan of the movies every few days. But I might want to run a scan of the TV libraries several times a day.

I quickly realized that I could change the `config.json` file to only scan the libraries I wanted. But in order to automate this through crontab, I thought it would be nice to be able to call different `config.json` files at different times of the day.

Currently, I just copied the entire script folder and I'm doing it that way. So I have two entirely separate instances of PlexTraktSync. But this is more a band-aid, rather than a permanent fix. Eventually it would be nice to have one copy of the script, and the ability to use different config files depending on what scan is taking place.

**Comment by glensc** [comment]
duplicates:
- https://github.com/Taxel/PlexTraktSync/issues/4
- https://github.com/Taxel/PlexTraktSync/pull/70

yes, the current recommendation is just to run two separate instances.

**Comment by glensc** [comment, marked as answer]
implemented as command-line option:
- https://github.com/Taxel/PlexTraktSync/pull/191
